src/lib/blocs/remark/remark_bloc.dart

```diff
@@ -279,6 +284,143 @@ class RemarkCubit extends Cubit<RemarkState> {
                 .toList())));
   }
 
+  /// Create an empty grading table
+  void initGradingTable() {
```

Comment:
Welche Funktionalität ist hier gegeben?

Reply:
Artefakt aus Zeiten, in denen `gradingTable` noch kein Attribut von `Exam` war. Wird entfernt.